docs/requirements.txt

```diff
@@ -1,19 +1,30 @@
-celery
+# https://github.com/sphinx-doc/sphinx/issues/2796
```

Comment:
Is this comment only valid for line 27 `sphinx==6.2.1` ?

Reply:
Sure! Thanks for catch.